docs/contributing/plugin-tutorial.md

```diff
@@ -355,38 +356,38 @@ generateLens pId uri minImports (L src imp)
   | RealSrcSpan l <- src
   , Just explicit <- Map.lookup (srcSpanStart src) minImports
   , L _ mn <- ideclName imp
-  -- (almost) no one wants to see an explicit import list for Prelude
+  -- (Almost) no one wants to see an explicit import list for Prelude
   , mn /= moduleName pRELUDE
   = do
         -- The title of the command is just the minimal explicit import decl
     let title = T.pack $ prettyPrint explicit
-        -- the range of the code lens is the span of the original import decl
+        -- The range of the code lens is the span of the original import decl
         _range :: Range = realSrcSpanToRange l
-        -- the code lens has no extra data
+        -- The code lens has no extra data
         _xdata = Nothing
-        -- an edit that replaces the whole declaration with the explicit one
+        -- An edit that replaces the whole declaration with the explicit one
         edit = WorkspaceEdit (Just editsMap) Nothing
         editsMap = HashMap.fromList [(uri, List [importEdit])]
         importEdit = TextEdit _range title
-        -- the command argument is simply the edit
+        -- The command argument is simply the edit
         _arguments = Just [toJSON $ ImportCommandParams edit]
-    -- create the command
+    -- Create the command
     _command <- Just <$> mkLspCommand pId importCommandId title _arguments
-    -- create and return the code lens
+    -- Create and return the code lens
     return $ Just CodeLens{..}
   | otherwise
   = return Nothing
 ```
 
 ## Wrapping up
 
-There's only one haskell code change left to do at this point: "link" the plugin in the `HlsPlugins` HLS module.
-However integrating the plugin in haskell-language-server itself will need some changes in config files. The best way is looking for the id (f.e. `hls-class-plugin`) of an existing plugin:
-- `./cabal*.project` and `./stack*.yaml`: add the plugin package in the `packages` field
-- `./haskell-language-server.cabal`: add a conditional block with the plugin package dependency
-- `./.github/workflows/test.yml`: add a block to run the test suite of the plugin
-- `./.github/workflows/hackage.yml`: add the plugin to the component list to release the plugin package to hackage
-- `./*.nix`: add the plugin to nix builds
+There's only one Haskell code change left to do at this point: "link" the plugin in the `HlsPlugins` HLS module.
+However integrating the plugin in HLS itself will need some changes in configuration files. The best way is looking for the ID (f.e. `hls-class-plugin`) of an existing plugin:
```

Comment:
full name would be better?

Reply:
Are you referring to `HLS` here? I think the tutorial uses the abbreviated form quite often, it might be even more consistent now.

Reply:
I have introduced the HLS abbreviation in the first paragraph. I think the usage is consistent now.